index.js

```diff
@@ -0,0 +1,2 @@
+var renderMap = require('./scripts/renderMap.js')
```

Comment:
This actually looks like a great place to require the data and geoJSON. Then you could pass the geoJSON into `renderMap()` instead of the data being referenced from inside `renderMap.js`. Much cleaner!